### ISSUE TITLE
not loading utilities

### ISSUE THREAD

**Issue by cruzchue** [johnaparker/fplanck#6]
Dear John;

Thanks for providing a code to solve FP.  I installed fplanck via conda in python 3.5, and now I am trying to re-run one of your examples, namely the ratchet.py:

https://github.com/johnaparker/fplanck/blob/master/examples/ratchet.py

however, I got some errors.  The first error appeared after loading fplanck:

    from . import utility
    raise ValueError(f'input vector ({value}) does not have the correct dimensions (ndim = {ndim})')
    SyntaxError: invalid syntax

As a quick solution, I just commented out the lines 12-13 in utility.py; that is:

    #        if vec.size != ndim:
    #            raise ValueError(f'input vector ({value}) does not have the correct dimensions (ndim = {ndim})')

Then, I got a second error:

    class boundary(enum.Enum):
    """enum for the types ofboundary conditions"""
    reflecting = enum.auto()

    class boundary(enum.Enum):
    """enum for the types ofboundary conditions"""
    reflecting = enum.auto()
    periodic   = enum.auto()

    AttributeError: module 'enum' has no attribute 'auto'

In short, fplanck can not be loaded clean.   Am I missing something?, perhaps a mistake from my side?, any suggestion?

Regards;

Eduardo
